Destiny.ScrimTracker.Logic/Models/Guardian.cs

```diff
@@ -0,0 +1,9 @@
+namespace Destiny.ScrimTracker.Logic.Models
+{
+    public class Guardian
+    {
+        public string GamerTag { get; set; }
+        public int Elo { get; set; }
+        public double AverageEfficiency { get; set; }
```

Comment:
I don't think we should track these in the Guardian model. I think we should have a `GuardianELO` model, which I'll talk about more in a different text box.

Same with `GuardianEfficiency`